components/layout/App.js

```diff
@@ -48,11 +54,15 @@ const App = ({ events }) => {
 
 App.propTypes = {
   // eslint-disable-next-line react/forbid-prop-types
```

Comment:
please remove this line if you have already disabled this eslint warning for the entire file

Reply:
Noted and done!